<launch>
	<group>
		<node pkg="iq_gnc" type="spawn.py" name="spawn1" output="screen" ns="/drone1">
			<param name="namespace" value="/drone1"/>
			<param name="takeoff_height" value='3'/>
			<param name="use_sim_time"  value="true" />
		</node>
	</group>
	<group>	
		<node pkg="iq_gnc" type="spawn.py" name="spawn2" output="screen" ns="/drone2">
			<param name="namespace" value="/drone2"/>
			<param name="takeoff_height" value='4'/>
			<param name="use_sim_time"  value="true" />
		</node>
	</group>
	<group>	
		<node pkg="iq_gnc" type="spawn.py" name="spawn3" output="screen" ns="/drone3">
			<param name="namespace" value="/drone3"/>
			<param name="takeoff_height" value='3.5'/>
			<param name="use_sim_time"  value="true" />
		</node>
	</group>
	<group>	
		<node pkg="iq_gnc" type="spawn.py" name="spawn4" output="screen" ns="/drone4">
			<param name="namespace" value="/drone4"/>
			<param name="takeoff_height" value='4'/>
			<param name="use_sim_time"  value="true" />
		</node>
	</group>
</launch>
